src/ucp/api/ucp.h

```diff
@@ -1417,13 +1420,31 @@ typedef struct ucp_worker_address_attr {
     uint64_t              worker_uid;
 } ucp_worker_address_attr_t;
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @enum ucp_op_type_t
```

Comment:
`ucp_ep_cost_op_type_t`
btw, why not to go with just `ucp_ep_op_type_t` or `ucp_op_type_t`? Potentially it could be reused by some future API